src/cloud-api-adaptor/test/provisioner/provision.go

```diff
@@ -424,7 +424,7 @@ func (p *CloudAPIAdaptor) Delete(ctx context.Context, cfg *envconf.Config) error
 	}
```

Comment:
And here is the other... I would suggest squashing too.

Reply:
Squashed